appcli/cli_builder.py

```diff
@@ -125,6 +125,11 @@ def create_cli(configuration: Configuration, desired_environment: Dict[str, str]
         multiple=True,
         callback=extract_valid_environment_variable_names,
     )
+    @click.option(
```

Comment:
We should have some doco in the README about this flag. What it is, how you use it, when and why you would want to use it (e.g. cron, scripts)

Reply:
I've added a section that contains the options/commands available and included the new --tty/--no-tty flag there. The readme is getting a bit bloated so it may be worth splitting some of the sections out or styling to contract some of the sections.